src/components/Panels/PiePanels/PieGraphPanel.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, ComponentProps } from "react";
+import { Pie } from "@/components/Pie/Pie";
+import { Graph } from "@/components/Graph/Graph";
```

Comment:
Reusable component will be at `Panel` level not at `Graph` level.
<GraphPanel/>

```
<GraphPanel>
   	<Graph data={data}>
		<Legend position={"top"} alignment={"center"} />
		<Pie {...state}>{state.children && <div dangerouslySetInnerHTML={{ __html: state.children.toString() ?? "" }} />}</Pie>
	</Graph>
</GraphPanel>
```

`PieGraphPanel` would only be used once; so better to keep this markup in the page itself.
hope this makes sense